src/libs/ReportUtils.ts

```diff
@@ -5415,7 +5424,12 @@ function shouldReportBeInOptionList({
             !isSelfDM(report) &&
             !isSystemChat(report) &&
             !isGroupChat(report) &&
-            !isInvoiceRoom(report))
+            !isInvoiceRoom(report) &&
+            // We omit sending back participants for certain reports when searching for reports since they aren't needed to display the results and can get very large.
+            // So we allow showing rooms with no participants when searching for reports.
+            // In any other circumstances we should never have these reports with no participants in Onyx.
+            !isSearchingForReports &&
```

Comment:
What was the result of testing? Is `isSearchingForReports` needed? Or is it fine to has some chats with no participants appear if they are not searchable by participants?

Reply:
Woops sorry for missing this–it should be fine–I did some testing with the LHN and didn't find anything out of the ordinary. Removed